boards/Board.mk

```diff
@@ -138,6 +164,8 @@ endif
 
 clean:
 	$(call DELFILE, libboard$(LIBEXT))
+	$(call DELFILE, $(ETCSRC))
+	$(call DELDIR, $(ETCDIR))
```

Comment:
clean RCOBJS here?

Reply:
RCOBJ is put into ETCDIR, so remove $(ETCDIR) is enough.